Cargo.lock

```diff
@@ -2337,29 +2299,6 @@ version = "0.3.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "121c2a6cda46980bb0fcd1647ffaf6cd3fc79a013de288782836f6df9c48780e"
 
-[[package]]
-name = "tower-lsp"
-version = "0.20.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "d4ba052b54a6627628d9b3c34c176e7eda8359b7da9acd497b9f20998d118508"
-dependencies = [
- "async-trait",
- "auto_impl",
- "bytes",
- "dashmap 5.5.3",
- "futures",
- "httparse",
- "lsp-types",
- "memchr",
- "serde",
- "serde_json",
- "tokio",
- "tokio-util",
- "tower",
- "tower-lsp-macros 0.9.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "tracing",
-]
-
 [[package]]
 name = "tower-lsp"
```

Comment:
Note how we drop the transitive dep on tower-lsp that we got through biome-lsp-converters

We now currently only depend on the `lionel-/tower-lsp` fork we have, which we will drop when we use lsp-server, also allowing us to drop all tokio dependencies